src/scripts/main.js

```diff
@@ -1,3 +1,38 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function handleClick() {
+  const promise1 = new Promise((resolve) => {
+    resolve('Promise was resolved!');
+  });
+
+  promise1.then(handleSuccess).catch(handleError);
+}
```

Comment:
The handleClick function is declared before the handleSuccess and handleError functions, which are used inside handleClick. It's a good practice to declare functions before they are used to make the code more readable and maintainable.